envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotProperties.kt

```diff
@@ -154,6 +155,11 @@ class CanaryProperties {
     var headerValue = "1"
 }
 
+class TrafficSplittingProperties {
+    var zoneName = ""
```

Comment:
What is  zoneName? Is it GCP, or gcp/west and gcp/central?. If the second option then LoadBalancingProperties should have a list or map of trafficsplitting properties